backend/qemu.pm

```diff
@@ -152,6 +152,57 @@ sub can_handle {
     return;
 }
 
+sub save_memory_dump {
+    my ($self, $args) = @_;
+    my $rsp             = 0;
+    my $vmname          = $args->{name};
+    my $migration_speed = $args->{migration_speed} // "4096m";
+
+    $rsp = $self->handle_qmp_command(
+        {
+            execute   => "migrate_set_speed",
+            arguments => {value => $migration_speed}});
+
+    bmwqemu::diag("Trying to save a QEMU image");
+
+    $rsp = $self->handle_qmp_command(
+        {
+            execute   => "migrate",
+            arguments => {
+                uri => sprintf("exec:gzip -c > ulogs/%s-vm-memory-dump.gz", $args->{filename}),
```

Comment:
No. this feature is implemented because we don't expect the developer debugging the failure available. If she was, we would have preferred to just keep the VM running. 

And if the developer is not available, we have all time on earth to dump and upload. The CPU of the qemu process is spare (as the VM is frozen), so use it fully and free up the worker slot.